### PR TITLE
fix: add readme for ama-sdk packages

### LINKED ISSUE #99: [Bug]: missing readme
### Contact Details

_No response_

### Package name

core, generator-sdk, swagger-builder

### Package version

8.0.4

### Reproduction steps

Check the readme at:
- https://www.npmjs.com/package/@ama-sdk/swagger-builder
- https://www.npmjs.com/package/@ama-sdk/generator-sdk
- https://www.npmjs.com/package/@ama-sdk/core

### Current result

No readme

### Expected result

Meaningful readme

### Additional comments

_No response_

### PR DESCRIPTION
**Pull request by vscaiceanu-1a**
fixes #99

### CODE REVIEW

File: packages/@ama-sdk/generator-sdk/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+*   **core**: To (re)generate the SDK based on a specified Swagger spec
+
+For generating the `shell`, you can run:
+```shell
+yarn yo ./node_modules/@ama-sdk/generator-sdk/src/generators/shell
+```
+Make sure to have a `./swagger-spec.yaml` file at the root of your project. One could use the following `scripts` in `package.json`:
+```json
+    "resolve": "node -e 'process.stdout.write(require.resolve(process.argv[1]));'",
+    "generate": "yo $(yarn resolve @ama-sdk/generator-sdk/src/generators/core) --swaggerSpecPath ./swagger-spec.yaml --force",
```
Comment (kpanot) [inline packages/@ama-sdk/generator-sdk/README.md:27]: This is working only with Yarn2+.
Maybe it should be specified

File: packages/@ama-sdk/generator-sdk/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+Yeoman is required to use this generator:
+```shell
+# Install Yeoman, @ama-sdk/core and the SDK generator
+yarn add --dev yo @ama-sdk/generator-sdk @ama-sdk/core
+```
+
+## How to use?
+
+### Typescript SDK
+The typescript generator provides 2 generators:
```
Comment (kpanot) [inline packages/@ama-sdk/generator-sdk/README.md:16]: You forgot the `create` generator that is running the two others

File: packages/@ama-sdk/generator-sdk/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# SDK Generator
+
+This package provide a [Yeoman](http://yeoman.io) generator to create an SDK based on a DAPI swagger spec.
```
Comment (cpaulve-1A) [inline packages/@ama-sdk/generator-sdk/README.md:3]:
```suggestion
This package provides a [Yeoman](http://yeoman.io) generator to create an SDK based on a DAPI swagger spec.
```

File: packages/@ama-sdk/generator-sdk/README.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+```shell
+yarn yo ./node_modules/@ama-sdk/generator-sdk/src/generators/shell
+```
+Make sure to have a `./swagger-spec.yaml` file at the root of your project. One could use the following `scripts` in `package.json`:
+```json
+    "resolve": "node -e 'process.stdout.write(require.resolve(process.argv[1]));'",
+    "generate": "yo $(yarn resolve @ama-sdk/generator-sdk/src/generators/core) --swaggerSpecPath ./swagger-spec.yaml --force",
+    "upgrade:repository": "yo $(yarn resolve @ama-sdk/generator-sdk/src/generators/shell)",
+```
+Use `generate` to (re) generate your SDK based on the content of `./swagger-spec.yaml` and `upgrade:repository` to regeneate the structure of your project.
```
Comment (cpaulve-1A) [inline packages/@ama-sdk/generator-sdk/README.md:30]:
```suggestion
Use `generate` to (re) generate your SDK based on the content of `./swagger-spec.yaml` and `upgrade:repository` to regenerate the structure of your project.
```

**Review by cpaulve-1A — APPROVED** [review]

File: packages/@ama-sdk/generator-sdk/README.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
+# SDK Generator
+
+This package provides a [Yeoman](http://yeoman.io) generator to create an SDK based on a DAPI swagger spec.
```
Comment (mrednic-1A) [inline packages/@ama-sdk/generator-sdk/README.md:3]: based on an API swagger spec

**Review by kpanot — APPROVED** [review]

**Review by mrednic-1A — APPROVED** [review]

**Pull request merged**